docs/api-testing/load-testing.md

```diff
@@ -111,6 +122,11 @@ Click any of the measurement labels to remove them from the display.
 
 <img src={useBaseUrl('/img/api-testing/load-test-report-status-codes.png')} alt="Status codes graph" width="400"/>
 
+Once the test is completed, at the bottom of the report, "Failures" section will be shown. In this section, you will see all the most recent failures that happened during the test. The section shows one row for each test execution. Each row has the date and time, the number of failures, and a link to the specific report for that test. 
```

Comment:
@LoryApiFortress Does the Failures section still show even if there are no test failures?

Reply:
@lysannep no, the section is shown only when test is completed AND there are failures, otherwise it is not shown

Reply:
Ok, thanks for clarifying.